extension/content.css

```diff
@@ -235,7 +235,7 @@
 
 /* increase visibility of news item when someone stars a repo */
 .watch_started.simple .body {
-	min-height: 40px;
+	min-height: 55px;
```

Comment:
Because I had to move the text around with `position: absolute` to match the look of the other feed items. The markup is different for items with the `.simple` class.